pkg/kubectl/resource/builder.go

```diff
@@ -274,12 +275,48 @@ func (b *Builder) SelectAllParam(selectAll bool) *Builder {
 	return b
 }
 
+func hasNamesArg(args []string) bool {
+	if len(args) > 1 {
+		return true
+	}
+	newArgs := []string{}
+	for _, s := range args {
+		newArgs = append(newArgs, strings.Split(s, ",")...)
+	}
+	args = newArgs
+	for _, s := range args {
+		if !strings.Contains(s, "/") {
+			return false
+		}
+		tuple := strings.Split(s, "/")
+		if len(tuple) != 3 && isGroup(tuple[0]) {
+			return false
+		}
+	}
+	return true
+}
+
+func getResource(groupResource string) string {
+	if strings.Contains(groupResource, "/") {
+		return strings.Split(groupResource, "/")[1]
+	}
+	return groupResource
+}
+
+func isGroup(group string) bool {
+	_, err := latest.Group(group)
+	return err == nil
+}
+
 // ResourceTypeOrNameArgs indicates that the builder should accept arguments
 // of the form `(<type1>[,<type2>,...]|<type> <name1>[,<name2>,...])`. When one argument is
```

Comment:
It seems this comment is outdated. It accepts these:
pods/name1 rc/name2
pods,rc name1 name2